odl/phantom/geometric.py

```diff
@@ -195,9 +196,25 @@ def indicate_proj_axis(space, scale_structures=0.5):
 
     Examples
     --------
-    >>> space = odl.uniform_discr([0] * 3, [1] * 3, [8, 8, 8])
-    >>> phan = indicate_proj_axis(space).asarray()
-    >>> print(np.sum(phan, 0))
+    Phantom in 2D space:
+
+    >>> np.set_printoptions(edgeitems=4)  # make numpy print whole arrays
+    >>> space = odl.uniform_discr([0, 0], [1, 1], shape=(8, 8))
+    >>> phantom = indicate_proj_axis(space).asarray()
+    >>> print(phantom)
```

Comment:
You may laugh at this, but a ascii representation is quite cool here:

```python
import numpy as np
import odl

def imgascii(img):
    img = np.array(img, dtype=float)
    chars = ' .:-=+*#%@'
    img -= img.min()
    img = (1.0 - img/(img.max() + 1e-6)) * (len(chars)-1)
    return str('\n'.join(''.join((chars[int(i)]) for i in row) for row in img))

img = odl.phantom.shepp_logan(odl.uniform_discr([0, 0], [1, 1], [32, 32]), 
                              True)

asc = imgascii(img)
print(asc)
```

```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@      @@@@@@@@@@@@@
@@@@@@@@@  ##########  @@@@@@@@@
@@@@@@@ ###############  @@@@@@@
@@@@@@################### @@@@@@
@@@@@##################### @@@@@
@@@@##########@@@@@@@@##### @@@@
@@@ ########@@@@@@@@@@###### @@@
@@@########@@@@@@@@@@####### @@@
@@ #######@@@@@@@@@%****##### @@
@@####*###@@@@@@@@*******#### @@
@@#########@@@%##********#### @@
@@############*##********#### @@
@@##########@@@@##*******#### @@
@@ ########@@@@@@@#*****##### @@
@@@#########@@@@@@@@######## @@@
@@@ ##########@@@@@@@####### @@@
@@@@#############@@@####### @@@@
@@@@@##################### @@@@@
@@@@@@################### @@@@@@
@@@@@@@ ###############  @@@@@@@
@@@@@@@@@  ##########  @@@@@@@@@
@@@@@@@@@@@@@      @@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
```

we could add a util xD